src/compiler/crystal/interpreter/cast.cr

```diff
@@ -48,7 +48,9 @@ class Crystal::Repl::Compiler
         compatible_type = to.union_types.find! { |union_type| type_needing_cast.implements?(union_type) }
 
         # Get the type id of the "from" union
-        from_type_id = get_union_type_id(aligned_sizeof_type(from), node: node)
+        # FIXME: This looks fishy. The variable is never used, only referenced in another comment.
+        # Can we safely drop this entire line?
+        from_type_id = get_union_type_id(aligned_sizeof_type(from), node: node) # ameba:disable Lint/UselessAssign
```

Comment:
Any suggestions on this?

Reply:
Name of the def is misleading then, since it suggest it's return value is significant (`get_...`), perhaps changing it to `<verb>_get_...` would be a better option, like `put_get_...`?